extensions/media-preview/media/videoPreview.js

```diff
@@ -33,6 +33,9 @@
 	}
 	video.playsInline = true;
 	video.controls = true;
+	video.autoplay = settings.autoplay;
+	video.muted = settings.autoplay;
```

Comment:
should we mention that videos will be muted in the setting description?

Reply:
This line makes it so that whether the video is muted depends on the autoplay setting.
Is that intentional?

Reply:
Yes, it's intentional, what I found is that videos won't start playing automatically if not muted [Autoplay policy in Chrome](https://developer.chrome.com/blog/autoplay/). 
ok, will add a comment about that in the settings

Reply:
Added `on mute` to the description